community.html

```diff
@@ -38,19 +38,18 @@ <h2 class="alt-h3">Office Hours</h2>
         to give feedback, ask questions, get help, or just chat about ideas.
       </p>
 
-      <span class="addtocalendar atc-style-blue">
+      <span class="addeventatc atc-style-blue" title="Add to Calendar">
         <a class="atcb-link btn btn-outline">
           {% octicon calendar class:"mr-2" %} Add to Calendar
         </a>
 
-        <var class="atc_event">
-          <var class="atc_date_start js-office-hours-start-time"></var>
-          <var class="atc_date_end js-office-hours-end-time"></var>
-          <var class="atc_timezone">UTC</var>
-          <var class="atc_title">Probot Office Hours</var>
-          <var class="atc_description">Join us every Friday to give feedback, ask questions, get help, or just chat about ideas.</var>
-          <var class="atc_location">{{ page.office_hours_link }}</var>
-        </var>
+        <span class="start js-office-hours-start-time" data-format="YYYY-MM-DD HH:mm"></span>
```

Comment:
I just tried this in development and the event that was created is off by 4 hours for me. I'm in Central timezone.

Reply:
Ugh that's because `moment` is mutatable by default :(

https://github.com/koddsson/probot.github.io/blob/cc678824bc79fb2ef1c7a5d971e73e1fd8928a6e/assets/js/office-hours.js#L18

Reply:
Sorry spoke too fast, that might not be the issue. 👀 now.